features/git-town/town.feature

```diff
@@ -12,3 +12,14 @@ Feature: Show correct git town usage
          or: git town non-feature-branches [(--add | --remove) <branchname>]
          or: git town version
       """
+
+  Scenario Outline: Running outside of a git repository
+    Given I'm currently not in a git repository
+    When I run `<COMMAND>`
+    Then I see the "git-town" man page
+    And I don't see "fatal: Not a git repository"
```

Comment:
I think we should still check it in this case. Showing the man page and printing stuff out aren't mutually exclusive